front/src/applications/operationalStudies/views/Study.tsx

```diff
@@ -93,6 +91,19 @@ export default function Study() {
       { skip: !projectId || !studyId }
     );
 
+  const {
+    selectedItemIds: selectedScenarioIds,
+    setSelectedItemIds: setSelectedScenarioIds,
+    itemIds: scenariosList,
+    setItemIds: setScenariosList,
+    toggleSelection: toggleScenarioSelection,
+    deleteItems: deleteSelectedScenarios,
+  } = useMultiSelection<ScenarioWithCountTrains>((scenarioId) => {
+    if (selectedScenarioIds.length > 0 && projectId && studyId) {
+      deleteScenarioV2({ projectId, studyId, scenarioId });
+    }
```

Comment:
same here

Reply:
Same response than for the previous commentary